spec/integration/request_controller_spec.rb

```diff
@@ -83,4 +83,31 @@
 
   end
 
+  describe 'FOI officer uploading a reponse' do
+
+    let(:user) { FactoryGirl.create(:user, :name => "Awkward > Name") }
+    let(:request) { FactoryGirl.create(:info_request, :user => user) }
+    let(:officer) { FactoryGirl.create(:user) }
+
+    before do
+      allow(request.public_body).
+        to receive(:is_foi_officer?).
+          with(officer).
+            and_return(true)
+    end
+
+    it 'should render a message confirming the response has been published' do
+      message = "Thank you for responding to this FOI request! " \
+                "Your response has been published below, and a " \
+                "link to your response has been emailed to Awkward &gt; Name."
+      using_session(login(officer)) do
+        visit upload_response_path :url_title => request.url_title
+        fill_in(:body, :with => 'Additional information')
+        click_button("Upload FOI response")
+        expect(page.body).to include(message)
```

Comment:
IIRC, I switched because `have_content` didn't work with HTML in the string - I'll have another go (either escaping the HTML or using a different matcher should help - cheers for the feedback! Don't let me take the lazy option when there's a better one available :slightly_smiling_face: )